patpich/medusa
# feat: generated client-js and client-react

## What

Generated `client-js` and `client-react` to replace `medusa-js` and `medusa-react`.
Implementation of generated `client-types`, `client-js`, and `client-react` in the Admin.

## Why

Eliminate SDK menial maintenaince tasks using generative code strategies.

## Tests

* Have a local medusa-server running
* Run `yarn install`
* Run `yarn build`
* Move to `packages/admin-ui/ui`
* Run `yarn dev --force`
* In a browser, open the address shown in the console output
* Expect the Admin to run as if nothing had changed
